backend/apps/skills_matching/views/legacy_views.py

```
@@ -0,0 +1,55 @@
+    IssueSkillTagSerializer,
+    NewcomerFriendlinessScoreSerializer,
+    RecommendationSerializer,
+    SkillGapAnalysisSerializer,
+    SkillTagSerializer,
+)
+
+
+class ContributorProfileViewSet(viewsets.ModelViewSet):
+    queryset = ContributorProfile.objects.all()
+    serializer_class = ContributorProfileSerializer
+    permission_classes = [IsAuthenticated]
```

Ruff (0.16.0)

[warning] Mutable default value for class attribute

(RUF012)